client-js/app/elements/labrad-registry.html

```diff
@@ -8,15 +8,24 @@
     :host {
       display: block;
     }
-    #reg-menu {
+
+    .z-container {
+      background: white;
+      height: 100%;
+      margin: 0px;
+    }
+
+    .reg-menu {
```

Comment:
These might make more sense as as classes as they may be re-usable in other parts of the app, of course it would then make sense to remove the ```reg``` identifier. They're not selected anywhere in the code so they don't need to be ids